srcpkgs/Komikku/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'Komikku'
 pkgname=Komikku
-version=0.14.0
+version=0.15.0
 revision=1
 archs=noarch
-wrksrc=Komikku-v${version}
+wrksrc=${pkgname}-v${version}
```

Comment:
Why is this better?

Reply:
It isn't. I just did to be consistent with other packages in the repo.

Reply:
There are packages that use literal too.
Using $pkgname is not semantic in wrksrc, if we create variant of package, say Komikku-lts (not probable for app, but happens for libraries or databases and so on), it is still `Komikku` needed here.

Reply:
Alright my bad. I have changed it back. I'll keep this in mind from now on.